.ci/run-container-ci

```diff
@@ -25,7 +25,7 @@
 set -e
 set -x
 
-CONTAINER=shiftcrypto/firmware_v2:16
+CONTAINER=shiftcrypto/firmware_v2:17
```

Comment:
You might want to update `latest` tag on docker hub:

![image](https://user-images.githubusercontent.com/25405/85943499-c56c4f80-b930-11ea-9b83-5d38164f3d4b.png)

I've just accidentally pulled a pile of gigabytes without specifying a tag, which defaults to `latest`, and later realized it's not the image I wanted.

Alternatively, drop the `latest` tag from docker hub, but I'm afraid some clients rely on its presents to fetch a manifest of images and it might break pulling.